FootballStats/Podfile

```
@@ -0,0 +1,12 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'FootballStats' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for FootballStats
+pod 'Alamofire'
+pod 'SDWebImage', '~> 5.0'
```

```suggestion
    pod 'Alamofire'
    pod 'SDWebImage', '~> 5.0'
```

---

Tất cả thư viên cần khai báo version khi import
```suggestion
    pod 'Alamofire', '~> x.x.x'
```

---

@hlinh88 phần này vẫn chưa đc sửa